apps/core/test/pubsub/integration_webhook/incidents_test.exs

```diff
@@ -39,7 +39,7 @@ defmodule Core.PubSub.IntegrationWebhook.IncidentsTest do
       assert Enum.map(results, &elem(&1, 0))
              |> ids_equal(webhooks)
 
-      for {webhook, %{payload: %{action: "incident.upated", payload: payload}} = log} <- results do
```

Comment:
I feel like this test has to be broken if it didn't modify the associated  implementation

Reply:
I know it can be hard to get a dev environment set up, if you'd like I'd be happy to test it for you, or you can give it a stab @kianmeng

Reply:
@michaeljguarino Do help, thanks.

Reply:
ok i'll probably just merge it and fix it on trunk